base/operators.jl

```diff
@@ -1109,6 +1109,16 @@ true
 
 julia> !(19 in a)
 false
+
+julia> [1, 2] .∈ [2, 3]
+2-element BitArray{1}:
+ 0
+ 0
+
+julia> [1, 2] .∈ ([2, 3],)
+2-element BitArray{1}:
+ 0
+ 1
```

Comment:
I think this example needs some explanation to be comprehensible.

Reply:
Yeah, you're probably right. I suppose it's ok to have more than one separate block of Examples in the docstring?

Reply:
Ok, I added an explanatory sentence. Not sure if it's clear enough.